src/main/java/com/google/api/generator/engine/ast/IfStatement.java

```diff
@@ -65,7 +65,7 @@ public IfStatement build() {
       IfStatement ifStatement = autoBuild();
       Preconditions.checkState(
           ifStatement.conditionExpr().type().equals(TypeNode.BOOLEAN),
-          "If-condition must be a boolean-typed expression");
+          "If-condition must be a boolean/Boolean-typed expression");
```

Comment:
No need to update the comment, the existing one is clear enough to readers.

Reply:
Done.